### PR TITLE
Wigner function functionality: Fixed an error

### PR DESCRIPTION
**Pull request by markjeveritt**
Fix for an indentation error in code.

## Description

As above.

## Motivation and Context

@diego-plan9 @ChristopheVuillot I am very sorry but an indentation error somehow got into our code. This pull request provides a fix for it.  

## How Has This Been Tested?

We have downloaded the revised code to a fresh location from GitHub and testing locally.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: qiskit/tools/visualization.py [hunk 1]
```diff
@@ -571,87 +569,84 @@ def plot_wigner_function(state, res=100):
...
     m = cm.ScalarMappable(cmap=cm.seismic_r)
-    m.set_array([-Wmax, Wmax])
+    m.set_array([-w_max, w_max])
     plt.colorbar(m, shrink=0.5, aspect=10)
     plt.show()
 
-def plot_wigner_data(wigner_data, phis=None,
-                     thetas=None, method=None,
+
+def plot_wigner_data(wigner_data, phis=None, thetas=None, method=None,
```
Comment (diego-plan9) [inline qiskit/tools/visualization.py]: In a similar note to a comment above, can you check if the parameter `theta` is needed, as it seems to be unused?

File: qiskit/tools/visualization.py [hunk 2]
```diff
@@ -571,87 +569,84 @@ def plot_wigner_function(state, res=100):
...
                      text_out=None):
```
Comment (diego-plan9) [inline qiskit/tools/visualization.py]: Can you also document `text_out` in the docstring?

File: qiskit/tools/visualization.py [hunk 3]
```diff
@@ -661,20 +656,19 @@ def plot_wigner_data(wigner_data, phis=None,
...
                 method = 'point'
             else:
                 method = 'curve'
-        elif wigDim ==2:
-            method ='plaquette'
+        elif wig_dim == 2:
+            method = 'plaquette'
 
     if method == 'curve':
         plot_wigner_curve(wigner_data, xaxis=phis)
```
Comment (diego-plan9) [inline qiskit/tools/visualization.py]: In the next lines:

```
     elif method == 'state':
         wigner_function(wigner_data, text_out)
```

`wigner_function()` seems not to be a valid function name: can you double-check?

File: qiskit/tools/qcvv/tomography.py [hunk 4]
```diff
@@ -733,77 +731,71 @@ def build_wigner_circuits(Q_program, name, phis, thetas, qubits,
...
     points = len(phis[0])
-    
-    
+
     for point in range(points):
         label = '_wigner_phase_point'
         label += str(point)
-        circuit = Q_program.create_circuit(label, [qreg], [creg])
+        circuit = q_program.create_circuit(label, [qreg], [creg])
         c_index = 0
```
[the remark this answers is not in the document]
Comment (markjeveritt) [reply qiskit/tools/qcvv/tomography.py]: It is not needed - removing now

**Comment by markjeveritt** [comment]
@diego-plan9 

Thank you for responding to us so quickly.

Sorry about the PR being issued against a previous version of the master branch - I still am getting used to git and did not realise I had done this. Did not mean to cause extra work.

Thank you so much for the rewrite of markjeveritt:master branch - we can see it took some effort from you.

We have checked the issues mentioned on the inline comments and fixed them. In all cases tis was removing unused variables as you suggested.

We have updated and tested the Jupyter notebook. This now works agains updated libraries but note we were only able to confirm against the simulator but not the experiment as the experiments timed out.

File: qiskit/tools/visualization.py (hunk 3 above)
Comment (markjeveritt) [reply qiskit/tools/visualization.py]: we fixed the function name

@diego-plan9 did we get everything please?

**Comment by markjeveritt** [comment]
@diego-plan9 Ok I am sorry but I am still confused by git - please can you let me know if we need to anything? I think this is the right place to ask/act but please let me know if I have got not wrong.

**Comment by diego-plan9** [comment]
Thanks for the changes, Mark! They are looking on the right track at a first glance, but I have not done a full review yet - if something else is needed, we make sure to let contributors know by commenting on the PR. In general, it is not unusual that the reviews and the "overall life time" of a pull request spawns over the course of days, depending on the work load and other factors - but we will get to it eventually, no worries!

**Comment by markjeveritt** [comment]
@diego-plan9 Understood. I am beginning to get git now - thanks for your patience.

**Comment by diego-plan9** [comment]
It's looking good, thanks for the contribution, Mark! I'll merge this PR and we can continue discussion if needed on the PR for the tutorials - please note that PR might take longer to get merged, as some changes to that repository are in progress.

**Pull request merged**